platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,20 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, *args, **kwargs):
```

Comment:
The other viewsets in this file avoid using *args and **kwargs. Is it possible for us to only used named positional and keyword arguments for this signature? I'm not sure.